modules/core/src/test/java/org/apache/ignite/internal/metric/JmxExporterSpiTest.java

```diff
@@ -730,6 +735,105 @@ public void testScanQuery() throws Exception {
         }
     }
 
+    /**
+     *
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testIgniteKernal() throws Exception {
+        DynamicMBean mbn = metricRegistry(ignite.name(), null, IGNITE_METRICS);
+
+        assertNotNull(mbn);
+
+        int cnt = 36;
+
+        for (MBeanAttributeInfo info : mbn.getMBeanInfo().getAttributes()) {
+            cnt--;
+
+            assertFalse(F.isEmpty(info.getDescription()));
+            Object val = mbn.getAttribute(info.getName());
+
+            switch (info.getName()) {
```

Comment:
Can we rewrite this test with the more dumb fashion? 

```
assertEquals(36, mbn.getMBeanInfo().getAttributes().size());

assertFalse(F.isEmpty(mbn.getAttribute("fullVersion")));
assertFalse(F.isEmpty(mbn.getAttribute("copyright")));
```

And so on...

Reply:
done